Deploy 2023-06-16

ctuguinay opened the pull request:
Pics and CSV additions for deployment 2023-06-16.

Additionally, some slight changes in tests:
- Add YeonJoon to CSV valid initials test
- Add Pytest marks for specific test selection
- Reorganize testing for field pictures

aditya-uw reviewed (changes requested):
Hi @ctuguinay! Thanks for requesting me for review, there are a couple things to ask:

1) Could you make a note for the Audiomoth UBNA_001 that has the error files that we ran into those error files in the notes section?
2) I noticed the Foliage deployment look kinda different from previous deployments, did you change the Audiomoth placement for Foliage?

ctuguinay commented:
Hey @aditya-uw:

1. I added a note for error files and for checking the moisture for the Audiomoth associated with the error files.
2. The foliage deployment is slightly different for one of them in foliage, primarily due to my inability to wrap it around the usual large branch, which was due to me not having a longer thing available. This time, it's wrapped around a smaller branch next to the usual large branch. I added a note for remembering to bring larger bands (the ones you recently purchased) in a previous commit.

aditya-uw commented:
@ctuguinay I understand! It's good the Audiomoths are facing the same general direction still! Thanks for explaining and making the additions.
